sssom/util.py

```diff
@@ -21,6 +22,10 @@
 OBJECT_SOURCE = 'object_source'
```

Comment:
All of these should probably be in a separate python file just for static variables.

Reply:
Done! https://github.com/mapping-commons/sssom-py/pull/72/commits/eb884e4a543da7a0ad872569f2c711922281c8b9

Reply:
Where do you suggest these be?